common/rpc-directory/multi-process-RPC-directory.mjs

```diff
@@ -115,7 +115,7 @@ class MultiProcessRPCDirectory extends Directory {
         methodName: methodName,
         args: JSON.stringify(args)
       }, {deadline: new Date(Date.now() + 5000)});
-      return JSON.parse(ret.response);
+      return ret.response ? JSON.parse(ret.response) : null;
```

Comment:
Log an error for invalid JSON? It shouldn't happen.

Reply:
The `ret.response` would be empty if the (remote) function doesn't return anything, causing the `JSON.parse` to throw error.